### PR TITLE
chore: Remove `Makefile`

### PR DESCRIPTION
**Pull request by lluiscampos**
It should be generated by `CMake` and not checked in

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -1,452 +0,0 @@
-# CMAKE generated file: DO NOT EDIT!
```
Comment (oleorhagen) [inline Makefile:1]: Says it all :p

File: .gitignore [hunk 2]
```diff
@@ -30,3 +30,4 @@
 *.cmake
 /Testing
 /install_manifest*.txt
+Makefile
```
Comment (oleorhagen) [inline .gitignore:33]: How about: `*Makefile`?

**Review by oleorhagen — APPROVED** [review]

**Review by kacf — APPROVED** [review]

File: .gitignore (hunk 2 above)
Comment (lluiscampos) [reply .gitignore:33]: This project generates only one `Makefile`, so I'd rather be explicit.

Comment (oleorhagen) [reply .gitignore:33]: Yes, but what if you build out of source :)

Comment (lluiscampos) [reply .gitignore:33]: Right. But `Makefile` (without the `/` prefix) will apply to any directory, right? It won't apply to `somethingMakefile` but that is okey.

Comment (oleorhagen) [reply .gitignore:33]: Oh, haha. TIL

**Pull request merged**